js/download.js

```diff
@@ -76,33 +112,24 @@ $(function() {
             revisionsString += "<div class='links'>";
 
             _.each(filesWhichMatch, function(file, index, list) {
-                var fileName = file.name;
-                var splitArtifactType = file.type.split("-");
-                var GTMID = splitArtifactType[0].charAt(0).toUpperCase() + splitArtifactType[0].slice(1) + '-' + splitArtifactType[1].charAt(0).toUpperCase() + splitArtifactType[1].slice(1) + "_" + artifact.version;
-
-                if (file.name.indexOf("server") > 0) {
-                    revisionsString += "<span><a id=" + GTMID + " class='icon-download' href=" + masterURL + subDirectory + '/' + fileName + ">Server</a></span> + ";
-
-                } else if (file.name.indexOf("agent") > 0) {
-                    revisionsString += "<span><a id=" + GTMID + " class='icon-download' href=" + masterURL + subDirectory + '/' + fileName + ">Agent</a></span>";
+                var type = downloadType(file);
+                if (!type) { return; }
+                revisionsString += '<span><a id="' + GTMID(artifact, file) + '" class="icon-download" href="' + masterURL + subDirectory + '/' + file.name + '">' + type + '</a></span>';
+                if (type === "Server") {
+                    revisionsString += " + ";
                 }
             });
             revisionsString += "</div>";
 
             revisionsString += '<div>'
             _.each(filesWhichMatch, function(file, index, list) {
-                if (file.name.indexOf("server") > 0) {
-                    revisionsString += '<div class="sha1">';
-                    revisionsString += "<div> <b>Server MD5</b> " + file.md5sum + " </div>";
-                    revisionsString += "<div> <b>Server SHA1</b> " + file.sha1sum + "</div>";
-                    revisionsString += '</div>';
-
-                } else if (file.name.indexOf("agent") > 0) {
-                    revisionsString += '<div class="md5">';
```

Comment:
That make sense. I just pushed another commit to consolidate them into 'checksums'.